src/com/facebook/buck/android/AndroidPrebuiltAar.java

```diff
@@ -119,6 +121,12 @@ public Path getBinaryJar() {
     return prebuiltJar.getPathToOutput();
   }
 
+  @Nullable
+  @Override
+  public Path getPathToOutput() {
+    return unzipAar.getPathToOutput();
```

Comment:
The reason I went down this path is `JavaBinary` followed a similar pattern of first copying the input jar to `buck-out` before doing anything with it.